dist/Makefile

```diff
@@ -1,5 +1,5 @@
 RELEASE      := 0.$(shell date +%Y%m%d).$(shell git describe --always)
-VERSION	     ?= 2.6-dev
+VERSION	     ?= 2.7-dev
```

Comment:
Why do we want that?

Reply:
now that we branched to 2.6 we should change the version in master to next dev release. This should be in master only no need to backport this

Reply:
Can we make it something generic? If not 3.0 is next release.

Reply:
for packaging it will simply our process to know that next branch is currently under dev. This is also how we do it in Scylla.
Changed to 3.0-dev